app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -360,18 +369,26 @@ def send_failed_fraud_email(user, enrollment)
       # rubocop:disable IdentityIdp/MailLaterLinter
       UserMailer.with(user: user, email_address: email_address).in_person_failed_fraud(
         enrollment: enrollment,
-      ).deliver_later(**mail_delivery_params)
+      ).deliver_later(**mail_delivery_params(enrollment.proofed_at))
       # rubocop:enable IdentityIdp/MailLaterLinter
     end
   end
 
-  def mail_delivery_params
-    config_delay = IdentityConfig.store.in_person_results_delay_in_hours
-    if config_delay > 0
-      return { wait: config_delay.hours }
-    elsif (config_delay == 0)
-      return {}
-    end
-    { wait: DEFAULT_EMAIL_DELAY_IN_HOURS.hours }
+  def mail_delivery_params(proofed_at)
+    mail_delay_hours = IdentityConfig.store.in_person_results_delay_in_hours ||
+                       DEFAULT_EMAIL_DELAY_IN_HOURS
+    wait_until = proofed_at + mail_delay_hours.hours
+    return {} if mail_delay_hours == 0 || wait_until < Time.zone.now
+    return { wait_until: wait_until }
+  end
+
+  def normalize_usps_timestamp(usps_timestamp)
+    return unless usps_timestamp
+    # Parse timestamps eg 12/17/2020 033855 => Thu, 17 Dec 2020 03:38:55 -0600
+    # Note that the USPS timestamps are in Central Standard time
+    ActiveSupport::TimeZone['Central Time (US & Canada)'].strptime(
+      usps_timestamp,
+      '%m/%d/%Y %H%M%S',
+    ).utc.to_datetime
```

Comment:
I would convert timezones like this which gives an ActiveRecord::TimeWithZone

```suggestion
    ).in_time_zone('UTC')
```

Reply:
Updated in [1bc8433](https://github.com/18F/identity-idp/pull/7805/commits/1bc8433b1b85413d1e3cb4068f76e4115fa1b891)